File: fortran/fortran77/Java/Fortran77LexerBase.java
import org.antlr.v4.runtime.*;

public abstract class Fortran77LexerBase extends Lexer
{
    public Fortran77LexerBase(CharStream input)
    {
        super(input);
    }

    protected boolean IsColumnZero()
    {
	return this.getCharPositionInLine() == 0;
    }
}

File: fortran/fortran77/Dart/Fortran77LexerBase.dart
import 'package:antlr4/antlr4.dart';
import 'dart:io';
import 'dart:convert';

abstract class Fortran77LexerBase extends Lexer
{
    Fortran77LexerBase(CharStream input) : super(input)
    {
    }

    bool IsColumnZero()
    {
        return charPositionInLine == 0;
    }
}
